nms/app/views/login/LoginForm.tsx

```diff
@@ -16,30 +16,62 @@ import Card from '@material-ui/core/Card';
 import CardActions from '@material-ui/core/CardActions';
 import CardContent from '@material-ui/core/CardContent';
 import FormLabel from '@material-ui/core/FormLabel';
+import Grid from '@material-ui/core/Grid';
 import React from 'react';
 import Text from '../../theme/design-system/Text';
 import TextField from '@material-ui/core/TextField';
+
+import {AltFormField} from '../../components/FormField';
+import {colors} from '@material-ui/core';
 import {withStyles} from '@material-ui/core/styles';
 
 const ENTER_KEY = 13;
 const styles = {
+  capitalize: {
+    textTransform: 'capitalize',
+  },
   card: {
     maxWidth: '400px',
-    margin: '10% auto 0',
+    margin: '24px auto 0',
+    padding: '20px 0',
+  },
+  cardContent: {
+    padding: '0 24px',
   },
   input: {
     display: 'inline-flex',
     width: '100%',
     margin: '5px 0',
   },
   footer: {
-    padding: '18px 16px 16px',
-    float: 'right',
+    padding: '0 24px',
+  },
+  login: {
+    marginTop: '10%',
   },
   title: {
```

Comment:
This should be sufficient
![Screenshot 2022-07-26 at 10 13 38](https://user-images.githubusercontent.com/102796295/180957641-34a4fe5b-9339-49d8-a81b-c4a10b482e9f.png)

Reply:
it was when I used the Grid component but now I need to keep most of it

Reply:
I see, this is because the title and subtitle are spans :|